### PR TITLE
Updated roadmap for Jan. 2020

### PR DESCRIPTION
**Pull request by nemanja-da**
Updated development roadmap for docs.daml.com

CHANGELOG_BEGIN
Removed previous roadmap (September 2019)
Added new one for January 2020
CHANGELOG_END

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: docs/source/support/roadmap.rst [hunk 1]
```diff
@@ -1,45 +1,42 @@
...
 
-  A HTTP/JSON-based high-level API with querying capabilities
-
-- **Ledger Ops Tooling**
+- **Deployment Options**
 
-  Tooling for monitoring, logging and health checking ledgers
+  - ALPHA version of DAML on Corda
 
 - **Canton**
```
Comment (cocreature) [inline docs/source/support/roadmap.rst]: I thought we want to stop using the name “Canton” externally?

Comment (nemanja-da) [reply docs/source/support/roadmap.rst]: Didn't get any feedback on that :( 

@bame-da Any input?

Comment (shaul-da) [reply docs/source/support/roadmap.rst]: Still discussing that. Please change to "DAML Interoperability"
It's okay for now to keep Canton in the bullet points

Comment (hurryabit) [reply docs/source/support/roadmap.rst]: @nemanja-da You need to ping the Canton team explicitly. They work in a different repo and hence probably don't get notifications here. @daravep is the person to ping for things like this one.

Comment (nemanja-da) [reply docs/source/support/roadmap.rst]: @shaul-da Ratko sent me a message and asked me to remove Canton updates

@hurryabit Thnx! Did got it touch with the team eventually :)

**Pull request merged**